.github/workflows/run-googletest.yml

```diff
@@ -0,0 +1,33 @@
+# Run C++ tests using GoogleTest
+name: run-googletest
+# on specifies the build triggers. See more info at https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows
+on:
+# The default build trigger is to run the action on every push and pull request, for any branch
+  push:
+  pull_request:
+  # To run the default repository branch weekly on sunday, uncomment the following 2 lines
+  #schedule:
+    #- cron: '0 0 * * 0'
```

Comment:
It may be worth adding a CRON trigger, if FIMS has dependencies that we want to make sure are still compatible (e.g., tmb). This could be done later, as needed.